test/tests/components/ws-manager/prebuild_test.go

```diff
@@ -187,7 +187,7 @@ const (
 // - stop the regular workspace
 // - relaunch the regular workspace
 // - make sure the file foobar.txt exists
-func TestOpenWorkspaceFromPrebuildSerialOnly(t *testing.T) {
```

Comment:
no more need to run this in serial. Other PRs have already made fixes to this test to make it run reliably